src/auth/impersonated.ts

```diff
@@ -30,7 +30,7 @@ export interface ImpersonatedOptions extends RefreshOptions {
   /**
    * Scopes to request during the authorization grant.
    */
-  targetScopes?: string[];
+  targetScopes?: string | string[];
```

Comment:
Any way to implement this without changing this type?

Reply:
indeed that's possible by using `targetScopes: this.getAnyScopes() as string[],` when instantiating the `Impersonated` class (in src/auth/googleauth.ts line 519)

The `BaseExternalAccountClient` as well as the  `JWT` class implement the scopes attribute with `string | string[]; `, thats why I thought to make it consistent. Also in oder to use the `getAnyScopes()` helper directly  

Any concerns adopting the ImpersonatedOptions type ?

Reply:
Since this is a `public` property that's not a union type, this would technically be breaking for consumers that expect this to be an `Array`